components/git/security.js

```diff
@@ -43,6 +43,10 @@ const securityOptions = {
   'post-release': {
     describe: 'Create the post-release announcement',
     type: 'boolean'
+  },
+  finish: {
```

Comment:
```suggestion
  finalize
```

Maybe `finalize` or `cleanup`. Finish doesn't sound good 😄